components/immatriculations/summary.tsx

```diff
@@ -30,8 +30,10 @@ const ImmatriculationSummary: React.FC<IJustificatifs> = ({
             </a>
           ) : (
             <a href="#insee">
-              <b>Cessée</b> auprès de l’Insee, depuis le{' '}
-              {formatDateLong(uniteLegale.dateDebutActivite)}
+              <b>Cessée</b> auprès de l’Insee
+              {uniteLegale.dateDebutActivite
+                ? `, depuis le ${formatDateLong(uniteLegale.dateDebutActivite)}`
```

Comment:
IMO le formatDateLong devrait renvoyer `''` quand la date est undefined. Ca permettrait d'avoir qqchose de plus smooth à la lecture.

C'est du pinaillage

Reply:
Alors en re-réflechissant a ta question : 

Je crois que je préférais avoir une signature en `string | undefined` pour qu'à la lecture tu te doutes que le `undefined` est pour les cas de date manquante. 

Si la signature est uniquement `string` c'est plus cohérent, mais tu es obligé d'ouvrir la fonction afin de savoir comment sont gérées les dates manquantes

Reply:
Yes de toute facon c'était clairement du pinaillage. In fine ce n'est pas très important :)